core/src/main/java/com/eternalcode/core/command/configurator/config/SubCommand.java

```diff
@@ -0,0 +1,40 @@
+package com.eternalcode.core.command.configurator.config;
+
+import net.dzikoysk.cdn.entity.Contextual;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Contextual
+public class SubCommand {
+
+    public String name;
+    public boolean enabled;
+    public List<String> aliases = new ArrayList<>();
+    public List<String> permissions = new ArrayList<>();
```

Comment:
to prywatne powinno  być

Reply:
Nain ponieważ configi się nie generują ;)